### PR TITLE
Add CICA accounts to SSO

### PR DESCRIPTION
**Pull request by bagg3rs**

### CODE REVIEW

File: terraform/organizations-accounts-cica.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ locals {
 }
 
 resource "aws_organizations_account" "cica" {
-  name      = "CICA"
+  name      = "CICA Shared Services"
```
Comment (jakemulley) [inline terraform/organizations-accounts-cica.tf:7]: Can you confirm you've followed the [How do I change the name on my AWS account?](https://aws.amazon.com/premiumsupport/knowledge-center/change-organizations-name/) guide for this - you'll need to login to the AWS root user for this AWS account.

File: terraform/organizations-accounts-cica.tf [hunk 2]
```diff
@@ -45,10 +45,11 @@ resource "aws_organizations_account" "cica-development" {
 }
 
 resource "aws_organizations_account" "cica-test-verify" {
-  name      = "CICA Test & Verify"
+  name      = "CICA Production"
```
Comment (jakemulley) [inline terraform/organizations-accounts-cica.tf:48]: Can you confirm you've followed the [How do I change the name on my AWS account?](https://aws.amazon.com/premiumsupport/knowledge-center/change-organizations-name/) guide for this - you'll need to login to the AWS root user for this AWS account.

**Review by jakemulley — CHANGES_REQUESTED** [review]

File: terraform/organizations-accounts-cica.tf (hunk 1 above)
Comment (bagg3rs) [reply terraform/organizations-accounts-cica.tf:7]: Have forwarded that info thanks

File: terraform/organizations-accounts-cica.tf (hunk 2 above)
Comment (bagg3rs) [reply terraform/organizations-accounts-cica.tf:48]: Have forwarded info to team

File: terraform/organizations-accounts-cica.tf (hunk 1 above)
Comment (jakemulley) [reply terraform/organizations-accounts-cica.tf:7]: You'll need to remove the name change in this PR until the above is completed.

File: terraform/organizations-accounts-cica.tf (hunk 2 above)
Comment (jakemulley) [reply terraform/organizations-accounts-cica.tf:48]: You'll need to remove the name change in this PR until the above is completed.

**Review by jakemulley — CHANGES_REQUESTED** [review]

**Review by ben-al — APPROVED** [review]
LGTM

**Review by jakemulley — APPROVED** [review]

**Pull request merged**